app/models/miq_group.rb

```diff
@@ -60,15 +60,18 @@ def self.seed
     ldap_to_filters = filter_map_file.exist? ? YAML.load_file(filter_map_file) : {}
     root_tenant = Tenant.root_tenant
 
+    groups = where(:group_type => SYSTEM_GROUP).includes(:entitlement).index_by(&:description)
+    roles  = MiqUserRole.where("name like '%EvmRole-%'").index_by(&:name)
```

Comment:
Shouldn't be here only `....where("name like 'EvmRole-%'")` ? regard to access to hash by index beginning on 'EvmRole...' on line 68 ?